packages/flutter/test/material/text_field_test.dart

```diff
@@ -1208,7 +1208,7 @@ void main() {
       editable.getLocalRectForCaret(const TextPosition(offset: 0)).topLeft,
     );
 
-    expect(topLeft.dx, equals(399.0));
+    expect(topLeft.dx, equals(398.5));
```

Comment:
Does this have to be reverted as well now that the default has been reverted to 1.0?

Reply:
Yes, I will do that. Thank you!